src/assets/stylesheets/_social_cards.scss

```diff
@@ -221,3 +230,32 @@ $blueDarken: #0085ac;
     min-height: 157px;
   }
 }
+
+
+// --------------------------------------------------------------- //
+// scrivito_dark adaptions
+// --------------------------------------------------------------- //
+
+.scrivito_dark {
+  background: #3f3f3f;
+
+  .scrivito_detail_label span {
+    color: #eee;
+    text-shadow: 0 1px 1px rgba(0, 0, 0, 0.3), 0 0 4px rgba(0, 0, 0, 0.2);
+  }
+  .scrivito_detail_label span.headline { color: #aaa; }
+  .scrivito_notice_body { color: #aaa !important; }
+
+
+  .seo_card_input .input {
+    background: rgba(222, 222, 222, 0.2);
+    color: #eee;
+  }
+
+  [data-scrivito-editors-placeholder]:empty:not(:focus):before { color: #aaa!important;}
+
+  .seo_card_preview .card { background: rgba(222, 222, 222, 0.2); }
+  .seo_card_preview .card_text h5 { color: #ccc!important;}
+  .seo_card_preview .card_text p { color:#ddd!important;}
+  .seo_card_preview .creator { color: #5e86be;}
```

Comment:
* [ ] FYI This color fails the contrast check https://webaim.org/resources/contrastchecker/?fcolor=5E86BE&bcolor=3F3F3F. In other places (e.g. changed attributes labels) we use a different blue (`#698cbe`), which passes at least for large text https://webaim.org/resources/contrastchecker/?fcolor=698cbe&bcolor=3F3F3F